js/ongaku.js

```diff
@@ -82,12 +85,24 @@ window.addEventListener("load", function () {
 		} else {return;}
 	}
 
+// Function to display track name
+	function displayTrackName(replay=false) {
+		if (replay) {
+			$('#track-name').delay(100);
+			$('#track-name').show();
+		}
+		trackName.innerHTML = order[0].name;
+		$('#track-name').fadeOut(10000);
+	}
+
+
 	// function to play tracks
 	let order = shuffle(data);
 	function play () {
 		music.src = order[0].link;
 		music.play();
 		updateButton();
+		displayTrackName();
```

Comment:
It works well !
Just one last thing is that suppose the track name is half faded and you select next track, then the new name is displayed half-faded and starts to fade.